src/Localization/Localization/ref/Microsoft.Extensions.Localization.netcoreapp.cs

```diff
@@ -62,22 +60,3 @@ public partial class RootNamespaceAttribute : System.Attribute
         public string RootNamespace { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
     }
 }
-namespace Microsoft.Extensions.Localization.Internal
-{
-    public partial class AssemblyWrapper
-    {
-        public AssemblyWrapper(System.Reflection.Assembly assembly) { }
-        public System.Reflection.Assembly Assembly { [System.Runtime.CompilerServices.CompilerGeneratedAttribute] get { throw null; } }
-        public virtual string FullName { get { throw null; } }
-        public virtual System.IO.Stream GetManifestResourceStream(string name) { throw null; }
-    }
-    public partial interface IResourceStringProvider
```

Comment:
What's the replacement for this? Should we move this to a public namespace?

Reply:
To my thinking it doesn't need replacing. It's an internal interface whose only purpose was to give us a point to create fakes for our integration tests. If anyone was using this interface to use something other than a `ResourceManager` as the backing for their [ResourceManagerStringLocalizer](https://github.com/dotnet/aspnetcore/blob/master/src/Localization/Localization/src/ResourceManagerStringLocalizer.cs) I would advise them that they should instead create an entirely new implementation of `IStringLocalizer`, since `ResourceManagerStringLocalizer` makes lots of assumptions based on the behavior of `ResourceManager`.